src/js/controller/setup-steps.js

```diff
@@ -25,15 +25,15 @@ define([
             },
             LOAD_YOUTUBE : {
                 method: _loadYoutube,
-                depends: ['LOAD_PLAYLIST']
+                depends: ['LOAD_POLYFILLS']
             },
             LOAD_SKIN : {
                 method: _loadSkin,
-                depends: []
+                depends: ['LOAD_POLYFILLS']
             },
             LOAD_PLAYLIST : {
                 method: _loadPlaylist,
-                depends: ['LOAD_PLUGINS']
+                depends: ['LOAD_PLUGINS', 'LOAD_YOUTUBE']
```

Comment:
RSS Playlist need to be loaded before LOAD_YOUTUBE's `hasYoutube`.

`_completePlaylist` should be a new step here, and the `_.isString(playlist)` handle loading earlier on.